### ISSUE TITLE
Requiring unknown module "request"

### ISSUE THREAD

**Issue by marc-schwartz** [nolanlawson/pouchdb-async-storage#3]
After init of a clean react-native project and following the `npm install --save pouchdb-async-storage
` and `npm install --save pouchdb@pouchdb/pouchdb#for-react-native` steps, App fails with following error screen:
![image](https://cloud.githubusercontent.com/assets/1254400/7615196/fee14c20-f969-11e4-8878-90d609c2a37b.png)

**Comment by nolanlawson** [comment]
Yeah this is definitely not production-ready yet. This module is a work in progress still.

Thanks for reporting! And if you manage to fix it, please do provide a pull request! :smiley:

**Comment by marc-schwartz** [comment]
Got it. Thanks for replying, Nolan.

**Comment by geirman** [comment]
+1 I had the same issue when I tried it, so now I'm trying AsyncStorage directly. Any idea how one might view the contents of AsynStorage in the same way browser tools let me view the contents of IndexDB and WebSQL?

**Comment by nrw** [comment]
+1 same issue here.

**Comment by andpor** [comment]
+1 same issue, npm install pouchdb...require('request') in ajaxCore.js is failing...line #3. ajaxCore is in lib/deps

**Comment by simonexmachina** [comment]
I fixed that issue by changing `"request": "./lib/deps/ajax/request-browser.js"` to `"request": "./pouchdb/lib/deps/ajax/request-browser.js"` in `node_modules/pouchdb/package.json`. No idea why that works, but then I hit another issue.

**Comment by andpor** [comment]
Has anyone got this to work?
